contrib/packs/actions/workflows/install.yaml

```diff
@@ -25,6 +25,12 @@
       ref: "packs.load"
       params:
         register: "{{register}}"
+      on-success: "reload-aliases"
```

Comment:
I want to explicitly call this out. This enables direct deploys of ChatOps from ChatOps with the single command. 

The `packs.load` command won`t allow me to take an array of things to register, and registering `all` is not the behavior we want (because of potentially rogue rules).

This works, and is a bit of a hack. If this makes you feel dirty, and it probably should, I'd ask for some cycles thinking about why I cannot just `all` here and be done with it. Having values that may or may not be desired to be loaded in an `all` sort of defeats the purpose. Maybe change the meaning of `all` (sensors, actions, aliases, etc). I don't know. Thoughts appreciated.